aspnetcore/signalr/configuration.md

```diff
@@ -162,6 +162,31 @@ let connection = new signalR.HubConnectionBuilder()
     .build();
 ```
 
+::: moniker range=">= aspnetcore-3.0"
+
+Instead of a `LogLevel` value, you can also provide a `string` value representing a log level name (see below for a list). This is useful when configuring SignalR logging in environments where you don't have access to the `LogLevel` constants.
+
+```javascript
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/myhub")
+    .configureLogging("warn")
+    .build();
+```
+
+The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or lower levels on the table**, will be logged.
```

Comment:
```suggestion
The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed below it on the table**, will be logged.
```

Otherwise, this could be misinterpreted as saying log levels with lower severity than the minimum log level will be logged which makes no sense.